packages/affiliates/libs/synthPrices.js

```diff
@@ -9,7 +9,14 @@ module.exports = ({ web3, cryptowatchApiKey, tradermadeApiKey, decimals = 18 } =
     from = from / 1000;
     to = to / 1000;
     const priceFeed = await createReferencePriceFeedForFinancialContract(
-      winston.createLogger({ transports: [new winston.transports.Console()] }),
+      winston.createLogger({
+        transports: [
+          new winston.transports.Console({
+            level: "error",
```

Comment:
Why do you define `level: "error"`? I don't think this affects what gets sent to `stdErr`, which is defined in the `stderrLevels` key below. 

I think what you're doing here by setting `level: error` is only logging error or higher logs, which might be what you're intending but FYI

Reply:
Agreed. @daywiss why not just apply this to _all_ log levels?

Reply:
maybe my understanding is wrong, but i thought this would log all messages at level `error` or more critical, and suppress any info/debug logs.

Then in stderLevels i want any logs error and above to log to `error` and not `out`

Reply:
Ah then I think you're right. You're specifying you want want logs at error or higher